pkg/Microsoft.NETCore.Platforms/runtimeGroups.props

```diff
@@ -55,28 +55,33 @@
     <RuntimeGroup Include="linuxmint.18">
       <Parent>ubuntu.16.04</Parent>
       <Architectures>x64</Architectures>
-      <Versions>1</Versions>
+      <Versions>1;2;3</Versions>
+    </RuntimeGroup>
+    <RuntimeGroup Include="linuxmint.19">
+      <Parent>ubuntu.18.04</Parent>
+      <Architectures>x64</Architectures>
+      <Versions></Versions>
```

Comment:
No versions?  I wonder if we should be listing the versions as 18, 18.1, 18.2, then 19 (in a seperate item).  I can't recall why I put the major version in the include statement initially.  If it works without a diff move the the major versions to the version metadata.